framework/src/st_things/things_stack/framework/things_data_manager.c

```diff
@@ -2389,22 +2411,22 @@ char *dm_get_mnid()
 	return g_device->mnid;
 }
 
-char *dm_get_firmware_version(void)
+char *dm_get_firmware_version()
```

Comment:
Why did you remove void in parameter?

Reply:
Its already handle in another PR #2544 . Because of this I remove it.